### ISSUE TITLE
e.toISOString is not a function error

### ISSUE THREAD

**Issue by delcroip** [lhncbc/lforms#102]
From what I saw it is because the objDate in that function is neither a Date nor a String but an object with asStr (string) as attribute

https://github.com/lhncbc/lforms/blob/95f8f70478c18ca581545773344b91a278d4f0fb/src/lib/lforms/lhc-common-utils.js#L104

Proposed approach: handle the new case
```
  dateToDTMString: function(objDate) {
    if(typeof objDate === 'string') {
      objDate = this.stringToDate(objDate, true);
    }else if(typeof objDate === 'object'){
      if (typeof objDate.asStr === 'string')
        objDate = this.stringToDate(objDate.asStr, true);
    } 
    return objDate.toISOString();
  },
```

**Comment by plynchnlm** [comment]
That solution would work, but I think this bug is actually coming from fhirpath.js.  The Object from asStr is an internal structure and is not meant to be something fhirpath.js returns-- as I recall.  Anyway, if you want to modify your local copy of lforms as you suggested, I think that would get you unstuck faster than it will take us to fix fhirpath.js and update lforms (which we will do).

**Comment by plynchnlm** [comment]
It is not entirely clear what fhirpath.js should be returning for now()-- a string?  A JavaScript Date object?  We will need to look into it.  I think any change we make will be a breaking change.  Whatever we return should be documented somewhere.  Anyway, this is just a note to say the fix might not be immediate.

**Comment by plynchnlm** [comment]
Fixed in version 32.0.1 of lforms.

**Comment by plynchnlm** [comment]
I posted that a little prematurely.  32.0.1 is now available.
